packages/remix-cloudflare-pages/index.ts

```diff
@@ -1,8 +1,2 @@
-import { installGlobals } from "./globals";
-
-export { createCloudflareKVSessionStorage } from "./sessions/cloudflareKVSessionStorage";
```

Comment:
Although I agree this doesn't belong here, this will probably be a breaking change for anyone running remix on cloudflare. For now, we should probably just deprecate importing `createCloudflareKVSessionStorage` from here and tell them to get it from `@remix-run/cloudflare` instead.